timemachine/fe/topology.py

```diff
@@ -329,6 +331,38 @@ def parameterize_periodic_torsion(self, proper_params, improper_params):
         combined_potential = potentials.PeriodicTorsion(combined_idxs, combined_lambda_mult, combined_lambda_offset)
         return combined_params, combined_potential
 
+    def setup_chiral_restraints(self):
+        # setup chiral restraints
+        chiral_atoms = chiral_utils.find_chiral_atoms(self.mol)
+        chiral_bonds = chiral_utils.find_chiral_bonds(self.mol)
+
+        restraint_k = [1000.0]
+
+        chiral_atom_restr_idxs = []
+        for a_idx in chiral_atoms:
+            idxs = chiral_utils.setup_chiral_atom_restraints(self.mol, get_romol_conf(self.mol), a_idx)
+            chiral_atom_restr_idxs.extend(idxs.tolist())
+
+        chiral_atom_restr_idxs = np.array(chiral_atom_restr_idxs)
+        chiral_atom_potential = potentials.ChiralAtomRestraint(chiral_atom_restr_idxs).bind(restraint_k)
```

Comment:
Minor surprise: `ChiralAtomRestraint(idxs).bind(restraint_k)` accepts a length-1 list instead of a length-`len(chiral_atom_restr_idxs)` array.

Reply:
I thought about this too - but I couldn't think of any reason to have multiple force constants for the restraints. Do you have one in mind?

Reply:
fixed! the setup code still takes in a single force constant, but the `params` itself in the potentials is now a vector.